packages/react-vapor/src/components/tooltip/DateTooltip.tsx

```diff
@@ -0,0 +1,24 @@
+import * as moment from 'moment';
+import * as React from 'react';
+import * as _ from 'underscore';
+import {Tooltip} from './Tooltip';
+
+export type DateTooltips = moment.Moment | string;
+
+export interface DateTooltipsProps {
+    date: DateTooltips;
+    format?: ((date: moment.Moment) => string) | string;
+    tooltipFormat?: string;
```

Comment:
Because we thought that the element containing the date could be more custom, but the tooltip would be a more formal type of format (which are covered with .format('style') of moment). But if we see he need to make it possible to use a function, i don't mind changing it

Reply:
Makes sense to me, I was wondering if it would be better for consistency to have the same logic for the 2 formats.